google_compute_engine_oslogin/pam_module/pam_oslogin_admin.cc

```diff
@@ -55,25 +55,24 @@ PAM_EXTERN int pam_sm_acct_mgmt(pam_handle_t *pamh, int flags, int argc,
   std::stringstream url;
   url << kMetadataServerUrl
       << "users?username=" << UrlEncode(str_user_name);
-  string response = HttpGet(url.str());
-  if (response == "") {
+  string response;
+  if (!HttpGet(url.str(), &response) || response.empty()) {
     return PAM_SUCCESS;
   }
   string email = ParseJsonToEmail(response);
-  if (email == "") {
+  if (email.empty()) {
     return PAM_SUCCESS;
   }
```

Comment:
This pam module's control flag is set to optional. The return value has no impact on a user's ability to log in.